docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

```diff
@@ -14,7 +14,7 @@ A risk effect is a measure of how some variable (the risk factor) causally impac
 some outcome variable. 
 
 Generally, we will use risk effects models to represent 
-associations between two variables that are assumed to be causal and 
+a causal effect of one variable on another and 
```

Comment:
This could probably use a broader sweep through other pages, but I figured I might as well start with this one. Our risk effects have explicitly causal interpretations and we need to state that clearly throughout IMO